src/desktop/native/hardware/Ledger.js

```diff
@@ -66,7 +65,7 @@ class Ledger {
                 if (message && message.abort) {
                     this.removeListener(callbackSuccess);
                     ipc.removeListener('ledger', callbackAbort);
-                    reject(connectionError);
+                    reject(Errors.LEDGER_CANCELLED);
```

Comment:
Shouldn't this be rejected with an `Error` object because [here](https://github.com/iotaledger/trinity-wallet/pull/1101/files#diff-ac38395aefdae2edd45175f6f364bb25R465 ) you are checking on `message` ?

Reply:
This error is catched at https://github.com/iotaledger/trinity-wallet/blob/6013bc45a0e223c8eb01d2b01a4ceed3ec1961f4/src/shared/libs/iota/utils.js#L336 which returns an `Error` object